BGL/doc/BGL/NamedParameters.txt

```diff
@@ -114,6 +114,21 @@ as result of a partition algorithm; if it is not provided, this information is
 simply inaccessible.
 \cgalNPEnd
 
+\cgalNPBegin{minimum_number_of_segments} \anchor BGL_minimum_number_of_segments
```

Comment:
I just looked at boost, and they are using max instead of maximum so we should use min here.